configs/steam-deck-romate/userConfigurations.json

```diff
@@ -1203,6 +1203,22 @@
       "category":"PlayStation 3",
       "name":"PlayStation 3"
    },
+   {
+      "id": "12345678901234567",
```

Comment:
To be confirmed with repo mods/admins

Reply:
@brenoprata10 made rommate. I forget how to update it, as i haven't touched this in a while. Maybe he can chime in.

Reply:
All good 👍

Reply:
Can you explain how the id works?

Is it supposed to be a secret?

Reply:
The id is not being used

Reply:
Ok, so what I understood from what you said is that it is just a placeholder, great!

Thank you